packages/core/src/domain/configuration.ts

```diff
@@ -139,6 +142,10 @@ export function buildConfiguration(userConfiguration: UserConfiguration, buildEn
     configuration.trackViewsManually = !!userConfiguration.trackViewsManually
   }
 
+  if ('censorshipLevel' in userConfiguration) {
+    configuration.censorshipLevel = userConfiguration.censorshipLevel ?? configuration.censorshipLevel
```

Comment:
I don't think we should diverge from the current configuration handling here. Just keep whatever value the user provides.

```suggestion
    configuration.censorshipLevel = userConfiguration.censorshipLevel
```